castle/cms/browser/controlpanel/status.py

```diff
@@ -0,0 +1,55 @@
+from Products.Five.browser import BrowserView
+import subprocess
+import redis
+import celery
+import celery.bin.base
+import celery.bin.celery
+import celery.platforms
+
+
+class StatusView(BrowserView):
+
+    def docsplit(self):
+        retval = self.DocsplitSearch()
+        if retval[0]:
+            status = 'OK'
+            icon = '<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Docsplit: '
+            return '<span>' + icon + status + '</span>'
```

Comment:
It would be better to have the display logic appear in the page template. This view should return True or False (and, in the case of False, the error string).

Reply:
Same for the other view methods below

Reply:
Updated my page template so the logic is in there instead of my python file. Having trouble figuring out how to get the error string into the page template. Anyone know of any docs I could look at to help?